extension/package.json

```diff
@@ -584,36 +588,39 @@
         "lodash": "^4.17.21",
         "minimatch": "^3.0.4",
         "node-powershell": "^3.3.1",
+        "original-fs": "^1.1.0",
+        "semver": "^7.3.5",
         "time-stamp": "^1.1.0",
-        "xmldom": "^0.5.0",
         "txml": "^4.0.1",
-        "semver": "^7.3.5"
+        "xmldom": "^0.5.0"
     },
     "devDependencies": {
         "@types/adm-zip": "^0.4.34",
-        "@types/capitalize": "^1.0.2",
         "@types/escape-string-regexp": "^1.0.0",
         "@types/format-duration": "^1.0.1",
         "@types/glob": "^7.1.3",
         "@types/jdataview": "0.0.31",
         "@types/lodash": "^4.14.168",
-        "@types/minimatch": "^3.0.3",
+        "@types/minimatch": "^3.0.4",
         "@types/mocha": "^8.0.4",
-        "@types/node": "^12.11.7",
+        "@types/node": "^12.20.12",
         "@types/node-powershell": "^3.1.1",
         "@types/semver": "^7.3.5",
-        "@types/vscode": "^1.50.0",
+        "@types/vscode": "^1.56.0",
```

Comment:
this one gives an error when packaging vsix:
`ERROR  @types/vscode ^1.56.0 greater than engines.vscode ^1.50.0. Consider upgrade engines.vscode or use an older @types/vscode version`